docs/how-to/use-rewards-api.md

```diff
@@ -0,0 +1,135 @@
+---
+description: Use the rewards API to view validator rewards.
+sidebar_position: 13
+---
+
+# View block rewards
+
+Use the [rewards API](https://consensys.github.io/teku/#tag/Rewards) to view the validator rewards for specific blocks.
+
+:::tip
+
+Enable the rewards API with the [`--rest-api-enabled`](../reference/cli/index.md#rest-api-enabled) command line option.
+
+A [Swagger interface is also available](../reference/rest.md#enable-the-rest-api-service). Use the [`--rest-api-docs-enabled`](../reference/cli/index.md#rest-api-docs-enabled) command line option to enable the web interface.
+
+:::
+
+## Limitations
+
+The following limitations apply:
+
+- The rewards API currently supports the `altair` fork upgrade and later.
+
+- The rewards API relies on state and block data to retrieve the reward information, meaning you'll receive limited data if the beacon node being queried is not an archive node.
+
+- Querying from a beacon node in `prune` storage mode only returns information for the the `finalized` block, and nothing before that.
+
+## Impact of data storage modes
+
+:::tip
+
+You can change the [`data-storage-mode`](../reference/cli/index.md#data-storage-mode) without re-initializing your database.
+
+You can change the frequency that states are stored by specifying [`data-storage-archive-frequency`](../reference/cli#data-storage-archive-frequency), but it will only affect the state storage from the time that the change has been made. It will also directly impact the amount of disk space required by Teku.
+
+:::
+
+Consider using a beacon node with `archive` mode storage if you frequently call the rewards API on finalized data. However, this may produce slow results due to having to replay blocks due to the infrequent storage of states on disk (every 2048 slots by default).
```

Comment:
I'm relatively happy with this (i think i wrote it), but just for clarity, i guess my point was if you want to use this frequently it may be worth tuning your data storage so that you can access the data more quickly, by storing more states (at the cost of disk space)

There'd be factors at play, but users could choose to store more states by setting archive-freqency=256 or even 64 for example, and replaying a lot less blocks... it would come down to space available and performance requirement on that api...

Reply:
I added a paragraph to provide extra clarity. Hope that helps.